File: templates/java/service_register.java
package {{pkg_name}}.{{service_name}}.service;

import {{pkg_name}}.{{service_name}}.handlers.*;

public class {{service_name}}Register {
    public static void register(){
        {{service_name}}.register();
        {{#flutter_msg}}
        {{service_name}}_{{name}}.register();
        {{/flutter_msg}}
    }
}

File: templates/dart/service_register.dart

 import '{{service_dart_file_name}}.dart';
 {{#native_msg}}import '../handlers/{{service_dart_file_name}}_{{msg_dart_file_name}}.dart'; 
 {{/native_msg}}
 
 class {{service_name}}Register{
 
  static void register(){
      {{service_name}}.regsiter();
      {{#native_msg}}
      {{service_name}}{{name}}.regsiter();
      {{/native_msg}}
   }
 
 }